AWSPluginsCore.podspec

```diff
@@ -10,7 +10,7 @@
 $AMPLIFY_VERSION = '1.2.0'
 $AMPLIFY_RELEASE_TAG = "v#{$AMPLIFY_VERSION}"
 
-$AWS_SDK_VERSION = '2.15.0'
+$AWS_SDK_VERSION = '2.16.0'
```

Comment:
Given that 2.16.0 is already out, and there needs to be another release to support XCode 12, seems like this needs to be 2.17.0 (along with everywhere else that references 2.16.0) ?

Reply:
Yes, we'll update to 2.17.0, but I want to do it incrementally as noted in the PR discussion above. The idea is to modify the PR incrementally to track progress of the dependencies, rather than a breaking PR that can't access unreleased dependencies.